.github/dependabot.yml

```diff
@@ -3,8 +3,13 @@ updates:
 - package-ecosystem: npm
   directory: "/"
   schedule:
-    interval: daily
+    interval: monthly
+  ignore:
+    - dependency-name: "@types/node"
+    - dependency-name: "*"
+      update-types: ["version-update:semver-patch"]
```

Comment:
Maybe we can ignore `minor` because we use `^` in `package.json`
```suggestion
      update-types: ["version-update:semver-patch", "version-update:semver-minor"]
```